src/main/java/com/synopsys/integration/jira/common/rest/oauth1a/JiraOAuthHttpClient.java

```diff
@@ -116,6 +116,9 @@ private GenericUrl createGenericUrl(String url, Map<String, Set<String>> queryPa
         URIBuilder uriBuilder = new URIBuilder(url);
         queryParams.entrySet()
             .stream()
+            .filter(entry -> entry.getValue() != null && entry.getValue()
+                                                             .stream()
+                                                             .allMatch(StringUtils::isNotBlank))
```

Comment:
I'm not a fan of using `&&` in `Stream.filter(...)`. The way streams get evaluated is that each step (like filter) in the stream will be evaluate for each element anyway. There is no performance improvement, but there is a readability detriment.